CHANGELOG.md

```diff
@@ -11,9 +11,9 @@ This release introduces breaking changes around how contacts are merged in order
   - **Backwards-compatibility:** the package will only reference the old `CONTACT_MERGE_AUDIT` table and create `stg_hubspot__contact_merge_audit` if `hubspot_contact_merge_audit_enabled` is explicitly set to `true` in your root `dbt_project.yml` file.
   
 ## Under the Hood
-[PR #98](https://github.com/fivetran/dbt_hubspot_source/pull/98) applies the following changes:
-- Updates seed data to test new merging paradigm.
-- Ensures that all timestamp fields are explicitly cast as timestamps without timezone, as recent API changes also introduced inconsistent timestamp formats.
+- Updates seed data to test new merging paradigm. ([#98](https://github.com/fivetran/dbt_hubspot_source/pull/98))
+- Ensures that all timestamp fields are explicitly cast as timestamps without timezone, as recent API changes also introduced inconsistent timestamp formats.([#98](https://github.com/fivetran/dbt_hubspot_source/pull/98))
+- The `CONTACT`, `COMPANY`, `DEAL`, and `TICKET` staging models have been updated to ensure users enabling the `hubspot__pass_through_all_columns` will have all required columns.
```

Comment:
should we make a `bug fixes` section for this fix?

Reply:
As I was writing this I was thinking of this as well. I think this actually should just be a bug fix as opposed to an under the hood section. I will make the adjustment.

Reply:
Updated and also added a buildkite test for `hubspot__pass_through_all_columns: true`